libkpod/attach.go

```diff
@@ -0,0 +1,182 @@
+package libkpod
+
+import (
+	"fmt"
+	"github.com/docker/docker/pkg/term"
```

Comment:
The standard for the code is to separate the builtin imports from the github and freind imports.  Can you separate them by space

Reply:
done